### DISCUSSION TITLE
.EXE file

### DISCUSSION THREAD

**Discussion by westland** [mworion/MountWizzard4#318] [Q&A] [answered]
I had heard that there is an .EXE install file for Windows 10/11 here.   I didn't see one, and wondered if anyone could help me on this.   I did check at the PyPl site and found the compressed [mountwizzard4-3.1.0.tar.gz ] but the file appears to be corrupted (tar on GitBash couldn't uncompress, nor could 7-zip).   I don't see any options for running under Python either.

Any help is appreciated

**Comment by mworion** [comment, marked as answer]
Hi,
in the past for MW3 I did a exe version, for current (MW4) there is no EXE available. Some reasons for it: 
- startup time will exceed 60 seconds with an exe (as the technology is self extracting every time)
- security signing needs accounts (and costs me money for a free software). I will definitely keep MW4 free of cost
- had in the past even more installer problem the with the actual python scripts.
- have to do it for Windows, MacOS, Ubuntu and some more platforms -> high effort for a single maintainer.

For sure if the will be a technology out to get this done efficiently, I will reinvestigate. So far I visited 4 different solutions, which all went to the same conclusion.

Michel
